test/Microsoft.ML.Tests/Transformers/TextFeaturizerTests.cs

```diff
@@ -249,7 +249,7 @@ public void NgramWorkout()
                 .Read(sentimentDataPath);
 
             var est = new WordTokenizingEstimator(Env, "text", "text")
-                .Append(new ValueToKeyMappingEstimator(Env, "text", "terms"))
+                .Append(new ValueToKeyMappingEstimator(Env, "terms", "text"))
                 .Append(new NgramExtractingEstimator(Env, "terms", "ngrams"))
                 .Append(new NgramHashingEstimator(Env, "terms", "ngramshash"));
```

Comment:
>NgramHashingEstimator [](start = 28, length = 21)

Nor this?>

Reply:
I am not converting the NGramHashing Estimator on this PR. It will come in the next PR. 

---
In reply to: [248095509](https://github.com/dotnet/machinelearning/pull/2118#discussion_r248095509) [](ancestors = 248095509)